### ISSUE TITLE
additional field

### ISSUE THREAD

**Issue by the-programmerr** [International-Data-Spaces-Association/DataspaceConnectorUI#309]
hello, I have one question, when I changed the ontology.ttl for adding a field in an additional section, the label of the feature(text feature)is not visible in UI. Is there any solution to this issue?hello, I have one question, when I changed the ontology.ttl for adding a field in an additional section, the label of the feature(text feature)is not visible in UI. Is there any solution to this issue?

**Comment by SebastianOpriel** [comment]
Do you have enabled the feature? https://github.com/International-Data-Spaces-Association/DataspaceConnectorUI/blob/4eb81e95172ccf845e10327f4fd747aa2a8871f6/src/backend/.env#L1

**Comment by the-programmerr** [comment]
yes, I can, but the label in the UI is invisible, should I choose a specific identifier? where can I see the available feature ?

![image](https://user-images.githubusercontent.com/105416067/211508007-680488a9-abea-452f-87b0-b4303a87b17d.png)

**Comment by SebastianOpriel** [comment]
A documentation of the feature can be found here: https://github.com/International-Data-Spaces-Association/DataspaceConnectorUI#add-custom-attributes-to-resources

**Comment by the-programmerr** [comment]
hello again, 
the problem has not been solved. based on the document I have added a new feature in ontology.ttl
![image](https://user-images.githubusercontent.com/105416067/214235499-df05327a-abe9-4d17-90d4-8ce607a29aa9.png)
and ontologyconfig.json
![image](https://user-images.githubusercontent.com/105416067/214235604-9a69456e-edaa-4fa0-b686-70e653efecd4.png)
in the backend, I can see the new feature, but the label of the additional field is empty.

Do you have any idea how can I solve it?should I change anything in the ontologyLoader.js?
Thanks in advance for your help.
